scripts/build-package.js

```diff
@@ -0,0 +1,76 @@
+const glob = require('glob');
+const path = require('path');
+const npmRun = require('npm-run');
+const argv = require('yargs').argv;
+
+/**
+ * Runs the build-babel or build-webpack npm target for the specified package.
+ */
+function runBuildTask(options) {
+  const task = options.npmTask || 'build-babel';
+  const args = ['run', task];
+  if (options.watch) {
+    args.push('--', '--watch');
+  }
+
+  const child = npmRun.spawn('npm', args, {
+    cwd: __dirname,
+    env: Object.assign({}, process.env, {
+      PACKAGE: options.packageName,
+    }),
+  });
+
+  child.stdout.on('data', (data) => {
+    process.stdout.write(data);
+  });
+
+  child.stderr.on('data', (data) => {
+    process.stderr.write(data);
+  });
+
+  child.on('close', (code) => {
+    if (code) {
+      process.stderr.write(`${options.packageName} exited with code ${code}`);
+    }
+  });
+}
+
+/**
+ * Run the build for Component Playground, since it's a webpack build it
+ * requires all other components to be built first.
+ * @param packageNames List of package names
+ * @param watch Whether to apply watch argument to the running task
+ */
+function runBuildPlayground(packageNames, watch) {
+  packageNames
+      .filter(pkg => pkg !== 'react-component-playground')
+      .forEach(packageName => (
+          runBuildTask({
+            packageName,
+          })
+        ));
+  runBuildTask({
+    packageName: 'react-component-playground',
+    npmTask: 'build-webpack',
+    watch,
+  });
+}
+
+const targetPackage = argv._[0];
+const applyWatch = Boolean(argv.watch);
+
+glob('./packages/react-*', null, (err, files) => {
+  const allPackagesNames = files.map(f => path.basename(f));
+
+  if (!targetPackage || targetPackage === 'react-component-playground') {
```

Comment:
You shouldn't compile all other packages every time you bundle CP.

You need to ensure order when we build all of them `npm run build`, but other than that you don't want to re-build other packages when you run `npm run build react-component-playground` when working exclusively on the playground. Not to mention that some packages are not deps of the playground such as `react-cosmos` and `react-cosmos-webpack`.

Having three separate cases seems like the clean solution. Wdyt?